Tools/Scripts/update-safer-cpp-expectations

```diff
@@ -167,7 +167,7 @@ def is_expected_file(expected_file):
     if not issues:
         print(f'{expected_file} has no known issues!')
     else:
-        print('Follow this link for the latest results: https://build.webkit.org/#/builders/Apple-Sonoma-Safer-CPP-Checks\n')
+        print('Follow this link for the latest results: https://build.webkit.org/#/builders/Apple-Sequoia-Safer-CPP-Checks\n')
```

Comment:
Though it isn't as elegant of a URL, would something like this (using tags) be better for future proofing? https://build.webkit.org/#/builders?tags=%2BSafer&tags=%2BCPP

Reply:
That's a good idea! It's also still one click to the most recent results so I think it's reasonable.